permission_handler_platform_interface/lib/src/permission_handler_platform_interface.dart

```diff
@@ -81,7 +81,7 @@ abstract class PermissionHandlerPlatform extends PlatformInterface {
   ///
   /// This method is only implemented on Android, calling this on iOS always
   /// returns [false].
-  Future<bool> shouldShowRequestPermissionRationale(Permission permission) {
+  Future<bool?> shouldShowRequestPermissionRationale(Permission permission) {
```

Comment:
This method should never return `null` and should be enforced here.
```suggestion
  Future<bool> openAppSettings() {
```